tests/lib/rules/require-default-props.js

```diff
@@ -1306,7 +1306,26 @@ ruleTester.run('require-default-props', rule, {
         column: 3
       }]
     },
-
+    //
+    // component with no pros followed by a failing component
+    {
+      code: [
+        'var ComponentWithNoProps = ({ bar = "bar" }) => {',
```

Comment:
updated the comment and fixed typo:

"component with no declared props followed by a failing component"